Misc/NEWS.d/next/Library/2023-08-05-08-41-58.gh-issue-107625.cVSHCT.rst

```diff
@@ -0,0 +1 @@
+Raise an :exc:`ParsingError` instead of an :exc:`AttributeError` when the configuration contains a key without a corresponding value in a scenario where extra space or a new line is present within the section.
```

Comment:
This note should mention what class or function it’s talking about, and be wrapped at 80 columns please.